source/fundamentals/crud.txt

```diff
@@ -9,4 +9,5 @@ CRUD Operations
 .. toctree::
    :caption: CRUD Operations
 
-   /fundamentals/crud/write-operations
\ No newline at end of file
+   /fundamentals/crud/write-operations
+   /fundamentals/crud/read-operations
```

Comment:
Issue: these sections don't have navigable landing pages from the TOC but you can navigate to them from the arrows that  bring you through all of the pages.

Suggestion: at least add a toc to these pages with links to the drawer pages.